src/utils/actions.ts

```diff
@@ -70,6 +71,11 @@ const EDITOR_ACTIONS : EditorAction[] = [
         keyBindings: [monaco.KeyMod.CtrlCmd | monaco.KeyMod.Shift | monaco.KeyCode.KeyC],
         label: "Copy Log Event",
     },
+    {
+        actionName: ACTION_NAME.WORD_WRAP,
+        keyBindings: [monaco.KeyMod.Alt | monaco.KeyCode.KeyZ],
+        label: "Toggle Word Wrap",
```

Comment:
we used to follow "Sentence case" for the action labels but i missed catching that in #197 . I think it's fine to use "Title Case" for this. if we want to unify the cases, we can submit a future PR.